easybuild/easyconfigs/g/Guile/Guile-3.0.8-GCCcore-11.3.0.eb

```
@@ -0,0 +1,45 @@
+easyblock = 'ConfigureMake'
+
+name = 'Guile'
+version = '3.0.8'
+
+homepage = 'https://www.gnu.org/software/guile/'
+
+description = """
+ Guile is a programming language, designed to help programmers create flexible
+ applications that can be extended by users or other programmers with plug-ins,
+ modules, or scripts.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['f25ae0c26e911af1b5005292d4f56621879f74d6958b30741cf67d8b6feb2016']
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('binutils', '2.38'),
+]
```

as per PR #16317 we should add a `pkg-config` (or rather, `pkgconf` if possible) build dependency
```suggestion
builddependencies = [
    ('Autotools', '20220317'),
    ('binutils', '2.38'),
    ('pkgconf', '1.8.0'),
]
```